pinot-common/src/main/java/org/apache/pinot/common/restlet/resources/EndReplaceSegmentsRequest.java

```diff
@@ -0,0 +1,42 @@
+package org.apache.pinot.common.restlet.resources;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Collections;
+import java.util.List;
+import javax.annotation.Nullable;
+
+/**
```

Comment:
Can we move this header to the top of the code? (above the `package org.apache.pinot....`

Reply:
> Can we move this header to the top of the code? (above the `package org.apache.pinot....`

ah thanks for catching !